tooling-mp-jwt-auth-tool/src/test/resources/pki/gen-keys.sh

```diff
@@ -0,0 +1,59 @@
+function gen_key_pair() {
```

Comment:
Why there are generated keypairs in tooling module? If those are test resources, please move them to module containing MP tests.

Reply:
those keys are used in tests e.g. `RsaKeyHelperTest`

Reply:
So these keys and their generation are now duplicated too?

Reply:
@honza-kasik the script is the same but the generated keys are different and are used in different tests & modules

Reply:
@tommaso-borgato This means that we have to maintain same code on two places. This needs to be solved.

Reply:
@honza-kasik fixed